monasca-alarms/monasca_alarm_definition.py

```diff
@@ -441,6 +423,10 @@ def _get_parser():
                         default=_env('OS_PROJECT_ID'),
                         help='Defaults to env[OS_PROJECT_ID].')
 
+    parser.add_argument('--os-tenant-name',
```

Comment:
I am not so sure about that, `tenant-name` is basically deprecated in favor of `project-name`. That said, I think we shouldn't introduce that but remove it.

Reply:
done